src/homepage/Components/Banner.js

```diff
@@ -1,13 +1,13 @@
 import React from "react";
-import styles from "../Css/Banner.module.css";
+import styles from "../CSS/Banner.module.css";
 import { Container, Row, Col, Jumbotron, Image } from "react-bootstrap";
 import code from "../assets/code.svg";
 import girl from "../assets/girl.svg";
 import smallscreen from "../assets/smallscreen.png";
 
 export default function Banner() {
   return (
-    <Jumbotron className={styles.Jumbotron} fluid>
+    <Jumbotron id="home" className={styles.Jumbotron} fluid>
       <br />
```

Comment:
@harsha14gupta Why these many `<br />` statements? Can't have CSS for `padding-bottom` / `margin-bottom`?